installer/conf/omsagent.d/baseline.conf

```diff
@@ -0,0 +1,30 @@
+# Baseline plugins
+
+<source> 
+    type exec
+    tag oms.baseline_immediate_run
+    command /opt/microsoft/omsagent/plugin/omsbaseline -d /etc/opt/microsoft/omsagent/conf/omsagent.d 

```

Comment:
As we add a number of these "immediate run" configurations, I worry about the load on start of the agent. What's the CPU cost of the omsbaseline run? We may want to prefix the command with a sleep to balance load.